Assets/Scripts/ArenaManager.cs

```diff
@@ -72,13 +84,20 @@ private void EndEpisode()
 
         private void ResetArena()
         {
-            print("Resetting");
             currentStep = 0;
-            for (int i = 0; i < teams.Count; i++)
+            for (int i = 0; i < teams.Length; i++)
             { 
                 teams[i].ResetTeam(i);
             }
             foodSpawner.ResetFood();
         }
+
+        private void SetTeamColors()
+        {
+            foreach (var team in teams)
+            {
+                team.SetTeamColor(UnityEngine.Random.ColorHSV(0f, 1f, 1f, 1f, 0.1f, 1f));
```

Comment:
Can we use something less random instead?  Something like... (you can tweak variables a little)

```csharp
        private Color NextColor(int count)
        {
            var shift = MAX_TEAMS / 2;
            float saturation = 1f;
            float value = 1f;
            if (count > (2 * MAX_TEAMS / 3))
            {
                shift += 1;
                saturation = 0.3f;
                value = 0.6f;
            }
            else if (count > (MAX_TEAMS / 3))
            {
                saturation = 0.5f;
            }
            count = (count * 5 + shift) % MAX_TEAMS;
            float hue = (float)count / MAX_TEAMS;
            return Color.HSVToRGB(hue, saturation, value);
        }
```

Reply:
Will included in upcoming improvements